mix.exs

```diff
@@ -43,7 +43,7 @@ defmodule Guardian.Mixfile do
 
   defp deps do
     [{:jose, "~> 1.8"},
-     {:phoenix, "~> 1.2.0", optional: true},
+     {:phoenix, "~> 1.2.0 or ~> 1.2-rc", optional: true},
```

Comment:
Should this be `1.3-rc` and will this require us updating the package for each release candidate?